docs/_inc/_create-classic-ui-instance.md

```diff
@@ -15,4 +15,4 @@ To stop the Plone instance in foreground mode, type {kbd}`ctrl-c`.
 
 ## Related content
 
-- {doc}`Classic UI </classic-ui/index>`
\ No newline at end of file
+- {ref}`Classic UI <classic-ui-index-label>`
```

Comment:
All of these can be simplified. Please remove the arbitrary text and let the reference do its magic to automatically generate the text when rendered. You can use `make livehtml` to see the changes build and reload automatically whenever you edit the file and save it.

```suggestion
- {ref}`classic-ui-index-label`
```